forest/src/cli/config.rs

```diff
@@ -5,18 +5,30 @@ use forest_chain_sync::SyncConfig;
 use forest_libp2p::Libp2pConfig;
 use forest_networks::ChainConfig;
 use serde::{Deserialize, Serialize};
-use std::sync::Arc;
+use std::{collections::HashSet, sync::Arc};
 
 use super::client::Client;
 
-#[derive(Serialize, Deserialize, PartialEq, Default)]
+#[derive(Serialize, Deserialize, Default, Clone, PartialEq)]
+pub struct LogConfig {
+    pub log_values: HashSet<LogValue>,
+}
+
+#[derive(Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
+pub struct LogValue {
+    pub module: String,
+    pub level: String,
+}
+
+#[derive(Serialize, Deserialize, Default, PartialEq)]
 #[serde(default)]
 pub struct Config {
     pub client: Client,
     pub rocks_db: forest_db::rocks_config::RocksDbConfig,
     pub network: Libp2pConfig,
     pub sync: SyncConfig,
     pub chain: Arc<ChainConfig>,
+    pub log_config: LogConfig,
```

Comment:
I would just use `log` and not `log_config`, otherwise it's a bit redundant.
Gossamer use that [too](https://chainsafe.github.io/gossamer/usage/configuration/#configuration).